assets/less/appeals/dec24.less

```diff
@@ -0,0 +1,397 @@
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  #appeal-header {
+
+    position: relative;
+    /* left: 50px; */
+
+    display: flex;
+    flex-direction: column-reverse;
+    gap: 30px;
+    margin: 0 auto 52px;
+    padding: 0 24px 24px;
+
+    width: 100%;
+    max-width: 900px;
+    align-items: center;
+    font-size: 1.25rem;
+
+    @media (min-width: @ss) {
+      gap: 60px;
+    }
+
+    @media (min-width: @sm) {
+      gap: 60px;
+      padding: 0;
+    }
+
+    @media (min-width: @md) {
+      padding: 0 24px 24px;
+      flex-direction: row;
+      gap: 9px;
+      align-items: flex-end;
+    }
+
+
+    & header {
+
+      /* Disable inherited styles */
+      background: none;
+      padding-top: 0;
+      padding-bottom: 0;
+      position: relative;
+
+      // todo: find a better way to set height?
+      height: 10rem;
+
+      @media (min-width: @md) {
+        height: 15rem;
+        display: flex;
+        flex-direction: column;
+        justify-content: flex-end;
+      }
+    }
+
+
+    #appeal-heading {
+      --txt: #FEFFFF;
+      width: 100%;
+      text-align: left;
+      display: flex;
+      flex-direction: column;
+
+      text-transform: uppercase;
+      font-size: 2.0rem;
+      font-weight: 600;
+      color: var(--txt);
+
+      & > span {
+        display: block;
+        font-size: 3.5rem;
+        font-weight: 600;
+        line-height: 3.5rem;
+
+
+
+      }
+    }
+
+    #illustration {
+      width: 90%;
+      height: 10rem;
+      position: relative;
+      @media (min-width: @md) {
+        width: 80%;
+      }
+    }
+    #illustration > #roc {
+      position: absolute;
+      top: -100px;
+
+      @media (min-width: @ss) {
+        top: -120px;
+      }
+      @media (min-width: @sm) {
+        top: -180px;
+      }
+      @media (min-width: @md) {
+        top: -250px;
+        left: -69px;
+
+      }
+    }
+
+  }
+
+  main {
+    padding: 1em;
+    margin: 0 auto;
+    max-width: 787px;
+    position: relative;
+  }
+
+  #appeal-body {
+    font-size: 1.3125rem;
+    font-weight: 400;
+    line-height: 1.5;
+  }
+
+  .letter-container {
+    padding: 60px 60px 10px;
+    background: var(--color-appeal-main-bg);
+    border: 1px solid var(--color-appeal-bg-border);
+    border-radius: 0.75rem;
+    box-shadow: var(--shadow);
+    position: relative;
+    margin-bottom: 121px;
+
+
+    p {
+      margin: 0 0 1.96875rem; // 1.5 font-size
+    }
+
+    p:nth-child(2) {
+      margin-top: 3rem;
+    }
+  }
+
+  .letter-container:before {
+    content: '';
+    position: absolute;
+    background: var(--bg-appeal-glow);
+    inset: -4px;
+    margin: 8px 38px;
+    border-radius: 0.75rem;
+    filter: blur(50px);
+    opacity: 0.8;
+    z-index: -1;
+  }
+
+  .closing-text {
+    color: var(--color-footer-txt);
+  }
+
+
+
+  // Copy and paste (with minor tweaks) from previous appeal
+  // TODO: Centralize this!
+  .closing-text {
+    font-family: Metropolis, sans-serif;
```

Comment:
Font seems to be open sans this time around, can you update this and throw it on the body {} main class as well. (Pending Laurel's comments on figma)